tutorials/tutorial02b.jl

```diff
@@ -0,0 +1,67 @@
+## For this tutorial, we start making use of GLAbstraction and GLWindow
+import GLFW
+using ModernGL, GeometryTypes, GLAbstraction, GLWindow
+
+# Create the window. create_glcontext is essentially a wrapper for
+# GLFW.CreateWindow, GLFW.MakeContextCurrent, and GLFW.ShowWindow, but
+# you can supply hints as well.
+window = create_glcontext("Tutorial 02b",
+                          resolution=(1024,768),
+                          windowhints=[(GLFW.SAMPLES, 4)]) # use 4x antialiasing
+
+# Dark blue background
+glClearColor(0.0f0, 0.0f0, 0.4f0, 0.0f0);
+
+# Retain keypress events
+GLFW.SetInputMode(window, GLFW.STICKY_KEYS, GL_TRUE)
+
+# Define the vertices of the triangle
+vertices = Point3f0[(-1,-1,0), (1,-1,0), (0, 1, 1)]
+
+# Define the shaders. Note that we don't specify the location (that
+# will be handled by the Dict below)
+vertexshader = vert"""
+#version 330 core
+
+in vec3 vertexPosition_modelspace;
+
+void main(){
+  gl_Position.xyz = vertexPosition_modelspace;
+  gl_Position.w = 1.0;
+}
+"""
+
+fragmentshader = frag"""
+#version 330 core
```

Comment:
I assumed (probably incorrectly) that this specifies "what dialect is this shader written in?" and not "which version of GLSL is running?" That is, suppose there have been (or someday might be) some backwards-incompatibilities; then specifying that "this shader was written for 3.3.0" might mean that all future versions of GLSL can still run it, because they've been "hinted" appropriately about how to interpret the code that follows.

If this is wrong, I'm happy to change it.